packages/react/.storybook/webpack.config.js

```diff
@@ -50,6 +50,7 @@ const styleLoaders = [
     options: {
       includePaths: [path.resolve(__dirname, '..', 'node_modules')],
       data: `
+        $css--reset: false;
```

Comment:
Should we just set all of these?

```
$css--font-face: false;
$css--helpers: false;
$css--body: false;
$css--use-layer: false;
$css--reset: false;
$css--default-type: false;
$css--plex: false;
```

Reply:
I think since we're not using the default sass import anymore, it's unnecessary. The difference with the `css--reset` is that even when we're importing components individually, it'll get included.

Reply:
Is that not the same case as with the other modules? A good number has shared CSS output that is only emitted once due to `$imported-modules: ()` which isn't being shared across threads.

Reply:
Are you getting duplicate styles in dev that are resolved from adding these? I wasn't but didn't want to add them if I didn't know what they were preventing.

Reply:
The components themselves shouldn't be importing any of those that generate styles right (body, plex, reset)? They just are for reset.

Reply:
They'll show up any time the corresponding partial is included, for example UI Shell and Pagination both included css--helpers.

Good overview of them the different flags is [here](https://github.com/carbon-design-system/carbon/blob/f98b1f6805c9beac4da34cc7e055e4d619a307f0/docs/guides/sass.md#global-flags) and they all have similar behavior in that a component imports stuff that it depends on (in theory) and `import-once` was handling only emitting once.